tests/test_coverage.py

```diff
@@ -22,40 +22,28 @@
 from viaf import VIAFClient
 
 
+class MockProvider(IdentifierResolutionCoverageProvider):
```

Comment:
Again, the changes to this file are already in master, I don't know why they're showing up in this diff.